sktime/forecasting/base/adapters/_generalised_statsforecast.py

```diff
@@ -55,7 +55,7 @@ def _get_validated_statsforecast_params(self):
         sktime_default_params = self.get_param_defaults().keys()
         statsforecast_params = self._get_init_statsforecast_params()
 
-        for sktime_param in sktime_params.keys():
+        for sktime_param in list(sktime_params):
```

Comment:
```suggestion
        for sktime_param in sktime_params:
```

Is list necessary here?